cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/test_fields.js

```diff
@@ -0,0 +1,27 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+const testFields = (dataTable) => {
+  cy.get('@fields').then((fields) => {
+    if (Object.keys(fields).length > 0) {
+      const fieldTestList = {};
+      if (Object.keys(fields).length > 0) {
+        dataTable.rawTable.forEach((fieldName) => {
+          fieldTestList[fieldName] = fields[fieldName];
+        });
+        cy.testDataTestidList(dataTable, fieldTestList);
+      }

```

Comment:
allerede sjekke på linje 4
```suggestion
      dataTable.rawTable.forEach((fieldName) => {
        fieldTestList[fieldName] = fields[fieldName];
      });
      cy.testDataTestidList(dataTable, fieldTestList);
```